x-pack/elastic-agent/pkg/agent/cmd/container.go

```diff
@@ -85,20 +96,16 @@ The following actions are possible and grouped based on the actions.
   FLEET_SERVER_CERT_KEY - path to private key for certificate to use for HTTPS endpoint
   FLEET_SERVER_INSECURE_HTTP - expose Fleet Server over HTTP (not recommended; insecure)
 
-* Elastic Agent Fleet Enrollment
-  This enrolls the Elastic Agent into a Fleet Server. It is also possible to have this create a new enrollment token
-  for this specific Elastic Agent.
+* Preparing Kibana for Fleet
```

Comment:
Shouldn't that paragraph be the first? I assume the order is:

1. Enable fleet with Kibana
2. Start fist agent with fleet server
3. Start second agent for monitoring the product (e.g. nginx)

Reply:
I think that fleet has to be setup in Kibana through Elastic Agent is not the recommended path. This is convenient for testing or in Cloud. So that is why I put it last.

For the fleet-server in the Cloud case, the fleet-server is not needed and already setup. That is why I put it second. See PR comment.